include/cvc5/cvc5_proof_rule.h

```diff
@@ -2330,14 +2377,25 @@ const char* toString(PfRule id);
  * @param id The proof rule to write to the stream
  * @return The stream
  */
-std::ostream& operator<<(std::ostream& out, PfRule id);
+std::ostream& operator<<(std::ostream& out, ProofRule id);
 
 /** Hash function for proof rules */
-struct PfRuleHashFunction
+struct ProofRuleHashFunction
```

Comment:
For consistency, perhaps this should change to the style for the hash function used by Kinds here: https://github.com/cvc5/cvc5/blob/main/include/cvc5/cvc5_kind.h#L5676.

(This code is using the standard practice for internal enums, but now that it is external maybe it should follow that one).